bmlt-meeting-list.php

```diff
@@ -502,9 +502,14 @@ function getFormatsForSelect($all = false)
             } else {
                 $services = '&services[]='.$service_body_id;
             }
-            $results = $this->get_configured_root_server_request("client_interface/json/?switcher=GetSearchResults$services&get_formats_only");
+            if (empty($service_body_id)) {
+                $queryUrl = "client_interface/json/?switcher=GetFormats";
+            } else {
+                $queryUrl = "client_interface/json/?switcher=GetSearchResults$services&get_formats_only";
```

Comment:
`client_interface/json/?switcher=GetSearchResults&get_formats_only` is not a valid query for aggregator. When you first set root server to aggregator there is no service body selected and the plugin was failing to load. After its saved you can then select a service body.

Reply:
This dumps on new installs, right?  Pretty sever.  Probably worth a version bump.  Should I add that into the pull request?

Reply:
Yes on new installs or someone moving to aggregator for first time, I did a minor bump to 2.7.9